qa/common/gen_qa_ragged_models.py

```diff
@@ -351,6 +449,10 @@ def create_batch_input_models(models_dir):
         create_modelconfig(models_dir, 4, model_version, np.float32,
                            "tensorflow", "savedmodel")
         create_savedmodel_modelfile(models_dir, model_version, np.float32)
+    if FLAGS.onnx:
+        create_modelconfig(models_dir, 4, model_version, np.float32,
+                           "onnx", "onnx")
```

Comment:
@GuanLuo : can you confirm for ort backend and platform name is onnx

Reply:
Also can you please help me understand if the config needs any onnx specific changes? and also point me to some documentation where I can understand these requirements. Thanks!

Reply:
backend should be "onnxruntime", and the platform can be left empty as ORT backend doesn't require it for further distinguishment. I don't think ONNX requires specific changes.